src/main/webapp/i18n/de/course.json

```diff
@@ -78,7 +78,7 @@
                     "forbidden": "Der Kurzname muss mit einem Buchstaben beginnen und darf keine Sonderzeichen beinhalten."
                 }
             },
-            "instructorDashboard": "Instruktor Dashboard",
+            "instructorDashboard": "Instruktor Kurs Dashboard",
```

Comment:
`Instruktor` is not really used in German. Can we change it to `Dozent` (in this case it would be `Dozenten` which is plural) or to `Kursleiter`?

We would need to change this consistently in all places. I guess this PR would be a good opportunity ;-)

Reply:
Sure, thank you for the suggestion!